src/app/store.js

```diff
@@ -0,0 +1,12 @@
+import { configureStore } from '@reduxjs/toolkit';
+import rocketsReducer from './features/rocketsReducer';
+import missionsReducer from './features/missionsReducer';
```

Comment:
- [ ] Please import logger from redux-logger and add it to your middleware since it is a requirement stated in the project description. you can do so by adding `import logger from 'redux-logger';` 👍

Reply:
this is not a requierments but whatever, why did it take so long for the review?